src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -257,7 +264,7 @@ protected String generatePackageUrl(final Artifact artifact) {
     }
 
     protected Component convert(Artifact artifact) {
-        return modelConverter.convert(artifact, schemaVersion(), includeLicenseText);
+        return modelConverter.convert(artifact, schemaVersion(), includeLicenseText, externalReferences);
```

Comment:
this method is for dependencies, not project being built: do not set external references

Reply:
Fixed in 20287587e8123fabf00afdacf6ab1cba581459be.